catroid/build.gradle

```diff
@@ -102,6 +103,7 @@ android {
             ext.enableCrashlytics = false
         }
         release {
+            buildConfigField "boolean", "FEATURE_EMBROIDERY_ENABLED", "true"
```

Comment:
im pretty sure this isnt going to be included in the next (soon to come release) so please set false here.